subresource-loading.bs

```diff
@@ -605,18 +608,18 @@ To <dfn>find a matching web bundle registration</dfn> given [=request=]
 
    1. Let |rule| be |registration|'s [=web bundle registration/rule=].
 
-   2. If |url| doesn't meet a path restriction rule, given |registration|'s
-      [=web bundle registration/fetch entry=]'s [=web bundle fetch
-      entry/source=], then return null.
+   2. If |url|'s [=url/scheme=] is not "`uuid-in-package`", then
+
+      1. Let |allowed path| be the result of [=shortening=] |registration|'s
+         [=web bundle registration/fetch entry=]'s [=web bundle fetch
+         entry/source=].
 
-      Issue: Clarify path restriction rule in algorithm.
+      2. If |url| doesn't start with |allowed path|, then return null.
```

Comment:
Shouldn't we continue the registration loop instead of returning null?

Reply:
Ops! Fixed!